assets/rich_text_components_definitions.ts

```diff
@@ -177,7 +177,7 @@ export = {
     "is_block_element": true,
     "customization_arg_specs": [{
       "name": "svg_filename",
-      "description": "The custom svg diagram filename",
+      "description": "The Custom SVG Diagram Editor",
```

Comment:
Maybe drop "Custom" and just call it "The SVG Diagram Editor"?

Reply:
Done.